BookingQueueSubscriber/BookingQueueSubscriber.Services/MessageHandlers/NewParticipantHearingConfirmationHandler.cs

```diff
@@ -43,11 +49,25 @@ public async Task HandleAsync(NewParticipantHearingConfirmationEvent eventMessag
                 RandomPassword = newUser.Password,
                 Name = $"{message.FirstName} {message.LastName}"
             };
-
+            
             await _bookingsApiClient.UpdatePersonUsernameAsync(message.ContactEmail, message.Username);
-
             await _userService.AssignUserToGroup(newUser.UserId, message.UserRole);
             await _notificationApiClient.SendParticipantSingleDayHearingConfirmationForNewUserEmailAsync(request);
+            
+            var participant = conferenceResponse.Participants.Single(x => x.ContactEmail == message.ContactEmail);
```

Comment:
just wondering if we should check whether the conferenceResponse is null here, in case the conference hasn't been created by this point? I don't think the published events necessarily complete in sequence

Reply:
Yeah I was discussing with Murali, I suggested like a polling function here, but I think the retries on the queue should be sufficient.

Reply:
ah, I didn't think about the retries. I guess in this case maybe we shouldn't handle it being null, as it wouldn't resend the message and the participant wouldn't get updated

Reply:
No worries, Ive added a polling mechanism now anyway, so all good